packages/origin-ui-core/src/features/certificates/sagas.ts

```diff
@@ -288,6 +288,63 @@ function* requestPublishForSaleSaga(): SagaIterator {
     }
 }
 
+function* requestDepositSaga(): SagaIterator {
+    while (true) {
+        const action: IRequestPublishForSaleAction = yield take(
+            CertificatesActions.requestDepositCertificate
+        );
+        const exchangeClient: IExchangeClient = yield select(getExchangeClient);
+
+        if (!exchangeClient.getAllTransfers) {
```

Comment:
Wondering about this check. Why we need that?

Reply:
It left after my experiments with certificates. Deleted